.github/actions/frontend/runtime/magic_url/action.yml

```diff
@@ -23,9 +23,6 @@ runs:
         echo "github_token=${{ inputs.github_token }}"
         echo "migration=${{ inputs.migration_number }}"
         echo "magic_url_route=${{ inputs.magic_url_route }}"
-    - name: npm_build
```

Comment:
@Sivli-Embir Addresses [this comment](https://github.com/JupiterOne/.github/pull/90#discussion_r1304924129)

Reply:
addressed